docs/GettingStarted.md

```diff
@@ -613,6 +615,29 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | ``service_name`` | Service name used for `mongo` instrumentation | mongodb |
 | ``quantize`` | Hash containing options for quantization. May include `:show` with an Array of keys to not quantize (or `:all` to skip quantization), or `:exclude` with Array of keys to exclude entirely. | ```{ show: [:collection, :database, :operation]  }``` |
 
+### MySQL2
+
+The MySQL2 integration traces any SQL command sent through `mysql2` gem.
+
+```ruby
+require 'mysql2'
+require 'ddtrace'
+
+Datadog.configure do |c|
+  c.use :mysql2, options
+end
+
+client = Mysql2::Client.new(:host => "localhost", :username => "root")
+client.query("SELECT * FROM users WHERE group='x'")
+```
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | --- | --- |
+| ``service_name`` | Service name used for MySQL2 instrumentation | mysql2 |
```

Comment:
Minor non blocking but you could wrap the mysql2 with a ` sign.

Reply:
On it

Reply:
`done()`